front/public/locales/fr/translation.json

```diff
@@ -244,7 +244,7 @@
           "pick-entry_point-cancel": "Annuler la sélection du point de départ",
           "pick-entry_point-update": "Modifier le point de départ sélectionné",
           "pick-exit_point": "Outil de sélection du point d'arrivée",
-          "pick-exit_point-cancel": "Annuler la sélection du point d'arrivée",
+          "pick-exit_point-cancel": "Annuler la sélection du point d'arrivée.",
```

Comment:
No `.` needed here
```suggestion
          "pick-exit_point-cancel": "Annuler la sélection du point d'arrivée",
```

Reply:
It was added to match the source version (containing a final `.`).
I will remove both the source and the French version.